apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/PatientLinkBuilderTest.java

```diff
@@ -103,6 +103,28 @@ public void fullPageTest() {
     Assert.assertTrue(bundle.getLink().isEmpty());
     paging.addLinks(bundle);
 
+    Assert.assertNotNull(bundle.getLink(Constants.LINK_SELF));
+    Assert.assertNotNull(bundle.getLink(Constants.LINK_FIRST));
+    Assert.assertNull(bundle.getLink(Constants.LINK_NEXT));
+  }
+
+  @Test
+  public void fullPageTestWithPaging() {
```

Comment:
I think you should consider adding a unit test for the `distinct` issue, if you can. Not a hard requirement; just a suggestion.

Reply:
FYI: We took care of this on Zoom last Friday.